### ISSUE TITLE
Gateway Monitor - FromDisk.pbit is not working

### ISSUE THREAD

**Issue by Gujjugaming2k** [RuiRomano/pbigtwmonitor#14]
Have any buddy checked, look like its not working @RuiRomano any update when it will be fixed ?

**Comment by sachinv22** [comment]
Yeah it isn't working for me either. the problem is with the Requests query wherein it's throwing an error that "Cannot convert value '' of type Text to type Date." and the other problem is with the Mashup Container profile table.

**Comment by kieran-steele** [comment]
Likewise. Same error. "Cannot convert value '' of type Text to type Date." in the requests query

This is actually my first use, found this funnily enough as the Microsoft OnPremisesDataGatewayLogs.pit still has that error "DataFormat.Error: There were more columns in the result than expected. Details: Count=11" from years ago.

I had a quick look at the Files query. I notice it takes the date and does

= Table.TransformColumnTypes(#"Renamed Columns",{{"Date", type date}})
and then near the end of the query it does

= Table.TransformColumnTypes(#"Removed Columns",{{"Date", type date}, {"Folder", type text}, {"Content Type", type text}})

In step 3 of Files from disk there is a date modified in date format, and when it gets grouped and comes back out its text in step 8.

Only other thing I can see, is when the query fails, the last line of code is :

//SUMMARIZE(UNION(logsRequests, queriesRequests), [RequestId], "Date", MIN([Dat, [Time])

In looking to check the syntax is correct (and note Dat, not Date)

https://amitchandak.medium.com/power-bi-power-query-vs-dax-append-and-summarize-data-233f173d0839#:~:text=%23%22Grouped%20Rows%22-,DAX%20Solution,-In%20Dax%2C%20we

Maybe? its meant to be "Date", MIN(Date, [Time]) and in a few rows above its TimeId but I dont think thats it.

**Comment by nicedane** [comment]
In the 'Requests' I had success in updating the "Date" dax to

"Date", 
    VAR st = [Start]
    RETURN IF(
        ISBLANK(st),
        BLANK(),
        DATEVALUE(FORMAT(st, "yyyy-MM-dd"))
    )

I had an empty string in my [Start], the above code replaces this with blank.
